crates/build/src/lib.rs

```diff
@@ -159,8 +161,10 @@ pub struct BuildResult {
     pub build_artifact: BuildArtifacts,
     /// The verbosity flags.
     pub verbosity: Verbosity,
+    /// Image used for the verifiable build
+    pub image: Option<String>,
```

Comment:
Where can one specify it? i.e. how can a user override the image used? Imagine a new version of ink released, or rust, I should be able to use it without installing new cargo contract.

Reply:
I have not added the command yet because the image has not been published and there is no way to test the functionality until then. Once the image is published. I will add the argument to the command to specify the running image